lib/manager/github-actions/__snapshots__/extract.spec.ts.snap

```diff
@@ -1,5 +1,44 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`lib/manager/github-actions/extract extractPackageFile() extracts multiple action tag lines from yaml configuration file 1`] = `
+Array [
+  Object {
+    "currentValue": "v1.0.0",
+    "datasource": "github-tags",
+    "depName": "actions/bin/shellcheck",
```

Comment:
I don't think this is valid tags - perhaps needs lookupName? Also github.com/actions/bin doesn't exist either?

Reply:
Ha, that's fair. I'll update the examples to something more realistic and maybe add a path nesting limit.

Reply:
if it's a/b/c then the a/b represents the repo while c is file or folder name? And if there's multiple actions within one repo then they share the same tags? Ie tags are per repo and not per action

Reply:
Hmmm. While actions [technically supports repo subpaths](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#example-using-a-public-action-in-a-subdirectory), the recommendation is [to have a single action per repo](https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/about-actions#choosing-a-location-for-your-action). At least for this first pass of getting this working, I wouldn't support the path'd version as the way most people would likely use that is via `SHAs` and not tags. Or would require some sort of tag prefixing support.

Reply:
If it's technically possible then we should ideally support it. In this case the lookupName would be `a/b`

Reply:
I actually did end up doing this. The latest version of the regex will extract `owner/repo` and ignore the path.